### PR TITLE
Migrated and Split Armor in to parts

### PR DESCRIPTION
**Pull request by kaseyawolf2**
Moved a large chunk of the Armor into separate files based on Location for easier look up and editing

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -1,68 +1,4 @@
...
-    "symbol": "[",
-    "color": "brown",
-    "covers": [ "LEGS" ],
-    "coverage": 75,
-    "encumbrance": 25,
-    "warmth": 5,
-    "material_thickness": 3,
-    "flags": [ "BELTED" ]
-  },
+[
```
Comment (kolsurma) [inline data/json/items/armor.json]: linting error, extra spaces at end of line?

**Comment by kolsurma** [comment]
Go ahead and throw each of the jsons through the web linter: http://dev.narc.ro/cataclysm/format.html

That'll clean up any errors with spaces and what not.

**Comment by Coolthulhu** [comment]
In general, it's much easier to do it in parts rather than in one huge go. You edited the entire armor.json, someone edited it in the meantime, so now there are conflicts.
You can solve the conflicts manually by merging current master onto your branch (but if there are problems, you may need to do it more than once) or cherry pick the non-conflicting commits and PR those separately.

**Comment by kevingranade** [comment]
Please just use an editor that can search, splitting jsons based on arbitrary attributes doesn't really make finding things any easier in general and creates needless churn in the meantime.

**Pull request closed without merging**